.github/ISSUE_TEMPLATE/questions-help-support.md

```diff
@@ -9,6 +9,35 @@ assignees: ''
 
 <!-- Please write your question here. -->
 
+## Information for reproducibility (optional)
+
+<!-- If you ask how to use Optuna's functionality in your concrete codes, the minimal reproducible code and related info are really helpful to answer your question.  -->
+
+### Environment
+
+- Optuna version:
```

Comment:
[nits]
I'm wondering if putting a sample script like below would be helpful to collect consistent information.

```python
import platform

import optuna


print(f"OS: {platform.platform()}")
print(f"Python: {platform.python_version()}")
print(f"Optuna: {optuna.__version__}")
```